.github/workflows/linux.yml

```diff
@@ -150,71 +150,55 @@ jobs:
           echo "::set-output name=dir::$CCACHE_DIR"
           echo "::set-output name=today::$(date -I)"
           echo "::set-output name=yesterday::$(date --date=yesterday -I)"
+
       - uses:  actions/cache@v2
         id:    cache-ccache
         with:
           path: ${{ steps.prep-ccache.outputs.dir }}
           key:  ccache-linux-release-${{ steps.prep-ccache.outputs.today }}
           restore-keys: |
             ccache-linux-release-${{ steps.prep-ccache.outputs.yesterday }}
+
       - name: Log environment
         run:  ./scripts/log-env.sh
-      - name: Build FluidSynth libraries
-        run: |
-          set -x
-          sudo apt-get install -y python3-setuptools cmake 
-          sudo pip3 install meson ninja
-          pushd contrib/static-glib
-          make
-          popd
-          cd contrib/static-fluidsynth
-          GLIB_LIBS="$PWD/../static-glib/lib/libglib-2.0.a" \
-          GTHREAD_LIBS="$PWD/../static-glib/lib/libgthread-2.0.a" \
-          GLIB_CFLAGS="-I$PWD/../static-glib/include/glib-2.0" \
-          GTHREAD_CFLAGS="-I$PWD/../static-glib/include/glib-2.0" \
-          make
+
       - name: Inject version string
         run: |
           set -x
           git fetch --prune --unshallow
-          export VERSION=$(git describe --abbrev=4)
-          sed -i "s/AC_INIT(dosbox,git)/AC_INIT(dosbox,$VERSION)/" configure.ac
+          export VERSION=$(git describe --abbrev=5)
           echo "VERSION=$VERSION" >> $GITHUB_ENV
-      - name: Build
+
+      - name: Setup release build
         run: |
-          set -x
-          ./autogen.sh
-          FLUIDSYNTH_CFLAGS="\
-              -I$PWD/contrib/static-fluidsynth/include \
-              -pthread -I$PWD/contrib/static-glib/include/glib-2.0" \
-          FLUIDSYNTH_LIBS="\
-              $PWD/contrib/static-fluidsynth/lib/libfluidsynth.a \
-              $PWD/contrib/static-glib/lib/libgthread-2.0.a -pthread \
-              $PWD/contrib/static-glib/lib/libglib-2.0.a -lm" \
-          ./configure \
-              --enable-png-static \
-              CFLAGS="$FLAGS" \
-              CXXFLAGS="$FLAGS" \
-              LDFLAGS="$FLAGS $LINKFLAGS -flto=$(nproc)"
-          make -j "$(nproc)"
-          strip src/dosbox
+          meson setup \
+            -Dbuildtype=release \
+            -Ddefault_library=static \
+            -Db_asneeded=true \
+            -Db_lto=true \
+            -Duse_fluidsynth=false \
+            build
```

Comment:
Looks great!

Reply:
It does :)

Initially I had `-Doptimization=3` in here as well, but then Meson gave "warning": "hey, it's already enabled by default for buildtype=release" :)